examples/Atlas/urdf/door.urdf

```diff
@@ -204,15 +204,14 @@
       </material>
     </visual>       
 <collision>   
-      <origin xyz="-0.1920875 -0.060325 0"
-        rpy="0 0 0" />
+      <origin xyz="-0.1920875 -0.060325 0" rpy="0 0 0" />
       <geometry>
         <box size="0.00953 0.0889 0.0254"/>
       </geometry>
       <material name="">
         <color rgba="0 0 1 1" />
       </material>
-</collision> 
+  </collision> 
   </link>
 
 <joint

```

Comment:
i guess technically these should be type "revolute" instead of "continuous" if they have joint limits:
http://wiki.ros.org/urdf/XML/joint